### PR TITLE
Supporting VSF-1.12 + Upgrading Sylius to 1.8

### PR DESCRIPTION
**Pull request by senghe**
We have prepared the plugin to support VSF 1.12. Also Sylius version for the plugin is upgraded to 1.8. This pull request is prepared for 2.0.0 version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,10 +64,10 @@ Please also check [Vue Storefront requirements](https://docs.vuestorefront.io/gu
 
 | Package       | Version        |
 |:-------------:|:--------------:|
-| PHP           |  7.3 (>= 7.2)  |
+| PHP           |  7.4 (>= 7.3)  |
```
Comment (Codeweld) [inline README.md:67]: Are you sure the 7.3 >= is not valid?

**Review by Codeweld — APPROVED** [review]
Why there is so much redundant code (from previous commits)?

**Comment by Codeweld** [comment]
Ok, I see it's to master. My mistake, it's fine.

File: README.md (hunk 1 above)
Comment (senghe) [reply README.md:67]: I don't understand. For now we recommend 7.4, but it has to be greater than 7.3.

Comment (Codeweld) [reply README.md:67]: Ahhh ok. My bad, it's fine.

**Pull request merged**